_includes/google_analytics.html

```diff
@@ -0,0 +1,15 @@
+<!-- Digital Analytics Program roll-up, see the data at https://analytics.usa.gov -->
+<script src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js" id="_fed_an_ua_tag"></script>
+
+<script>
+  // Google Analytics
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-48605964-49', 'auto');
```

Comment:
@ericronne I think instead of `'UA-48605964-49'` here you want `{{ include.id }}` and then
set it in the `_config.yml`:

```
google_analytics:
  default: UA-48605964-49
```